website/docs/jellyfish/api/masternode.md

```diff
@@ -0,0 +1,127 @@
+---
+id: masternode
+title: Masternode
```

Comment:
There are missing wordshere. Please check

Reply:
The API right? Added.